keyboards/slothie60/readme.md

```diff
@@ -0,0 +1,26 @@
+# slothie60
+
+![slothie60]
```

Comment:
This is intended to be an [image](https://www.markdownguide.org/basic-syntax/#images).